File: 프로그래머스/lv1/42576. 완주하지 못한 선수/완주하지 못한 선수.java
import java.util.*;

class Solution {
    public String solution(String[] participant, String[] completion) {
        String answer = "";
        Arrays.sort(participant);
        Arrays.sort(completion);
        completion=Arrays.copyOf(completion,completion.length+1);
        completion[completion.length-1]="A";
        for(int i=0;i<participant.length;i++){
            if(!participant[i].equals(completion[i])){
                return participant[i];
            }
        }
        return answer;
    }
}

File: 프로그래머스/lv1/42576. 완주하지 못한 선수/완주하지 못한 선수.js
function solution(participant, completion) {
    participant.sort();
    completion.sort();
    completion.push("A");
    participant.reverse();
    completion.reverse();
    answer='';
    participant.forEach((a,b)=>{
        if(a!=completion[b]){
            answer=a;
        }
    });
    return answer;
}